frontend/coprs_frontend/coprs/logic/builds_logic.py

```diff
@@ -1498,13 +1498,18 @@ def new(cls, build, mock_chroot, **kwargs):
 
     @classmethod
     def get_by_build_id_and_name(cls, build_id, name):
-        mc = MockChrootsLogic.get_from_name(name).one()
+        try:
+            mc = MockChrootsLogic.get_from_name(name).one()
 
-        return (
-            BuildChroot.query
-            .filter(BuildChroot.build_id == build_id)
-            .filter(BuildChroot.mock_chroot_id == mc.id)
-        )
+            return (
+                BuildChroot.query
+                .filter(BuildChroot.build_id == build_id)
+                .filter(BuildChroot.mock_chroot_id == mc.id)
+            )
+        except DataError as ex:
+            raise MalformedArgumentException(
+                f"Invalid build_id: {build_id} or name: {name}"
+            ) from ex
```

Comment:
I'm not sure where the DataError comes from.  Do we *really* need this part at all?

Reply:
reproducer: https://copr.fedorainfracloud.org/backend/get-build-task/6742s605-fedora-39-aarch64

Reply:
Are you sure?  If I apply the change above, I'm getting this locally:
```
curl http://127.0.0.1:55555/backend/get-build-task/6742s605-fedora-39-aarch64
{"msg":"Specified task ID not found"}
```

Reply:
```
curl https://copr.fedorainfracloud.org/backend/get-build-task/871s1675-fedora-42-x86_64
```

produces in logs:

```
sqlalchemy.exc.DataError: (psycopg2.errors.InvalidTextRepresentation) invalid input syntax for type integer: "871s1675"
2025-03-17 16:32:04,543 [ERROR][/usr/share/copr/coprs_frontend/coprs/error_handlers.py:42|error_handlers:handle_error][ANON] Response error: 9h9h Database error, contact admin
```

the same goes for staging instance

Reply:
Right, I can reproduce now when f39 aarch64 exists.